### ISSUE TITLE
Homey Compose invalid driver flow template?

### ISSUE THREAD

**Issue by edwinvdpol** [athombv/apps-sdk-issues#218]
Hi there,

When I run the `homey app validate` CLI command, I get the following error:

`Invalid driver flow template for driver bridge: connected_false`

This always worked as a charm. Why is it not working anymore?

There was a fix [2 days ago](https://github.com/athombv/node-homey/commit/5e16a70e98c08b8e46fd17390a6fb32a271727c4), not sure if this solves the issue?
If so, please release 😉 

Homey version: 2.20.0
Node version: 16.3.0

**EDIT 2:** Wanneer ik [deze](https://github.com/athombv/node-homey/pull/202/files#diff-fdb0a9b0d140132f4af22a1a6764b1c1fa7499682805da5aea01ccd91d0933c2L498-R500) en [deze](https://github.com/athombv/node-homey/pull/202/files#diff-fdb0a9b0d140132f4af22a1a6764b1c1fa7499682805da5aea01ccd91d0933c2L297-R265) change revert, gaat hij wel door. Krijg daarna alleen wel de melding:
`Warning: found multiple Flow card triggers with the id "connected_false", all Flow cards should have a unique id.`

**EDIT:** I updated the file with the new one, but it doesn't fix the issue unfortunately.
I do get the flowcards, but it goes wrong [here](https://github.com/athombv/node-homey/blob/master/lib/HomeyCompose.js#L269)

<details><summary>Debug info</summary>
<p>

console.log resultaten van de variabelen...

**templateIds**
```
[ 'connected_false' ]
```

**templateCards**
```
{
  charging_false: { title: { en: 'A lock is charged', nl: 'Een slot is opgeladen' } },
  charging_true: {
    title: { en: 'A lock just charged', nl: 'Een slot werd opgeladen' }
  },
  connected_false: {
    title: {
      en: 'The connection was lost',
      nl: 'De verbinding werd verbroken'
    }
  },
  connected_true: {
    title: {
      en: 'The connection was restored',
      nl: 'De verbinding werd hersteld'
    }
  },
  opened: {
    title: { en: 'The lock just opened', nl: 'Een slot werd geopend' }
  },
  update_available_true: {
    title: { en: 'An update is available', nl: 'Een update is beschikbaar' }
  }
}
```

**templateId**
```
connected_false
```

**templateCards.templateId**
```
undefined
```

</p>
</details>

**Comment by tjallingt** [comment]
Thanks for reporting and investigating!
We recently fixed all ESLint errors in the Homey CLI and this bug was accidentally introduced.

**Comment by tjallingt** [comment]
A fix for this is released in Homey CLI version 2.20.2
